portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase

```diff
@@ -428,6 +428,56 @@ definition {
 		ACDXPSettings.connectDXPtoAnalyticsCloud();
 	}
 
+	@description = "Bug: LRAC-10629 | Automation ID: LRAC-11443 | Test Summary: Sites and Contacts connection status is not changing when renaming the data source name"
+	@priority = "3"
+	test ConnectionStatusNotChangeWhenRenameDataSourceName {
+		task ("Launch AC then go to Data Sources and open the created data source") {
+			var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud();
```

Comment:
Hey @manuferreira, just a small change here, it is not necessary to put the property name in this variable, the macro returns a global value for every test that uses the macro. So to access the property name you can use `${assignedPropertyName}`

Reply:
Yes, I like it. Updates on test and test map. Thank you, Manu